.changeset/large-buckets-camp.md

```diff
@@ -0,0 +1,5 @@
+---
+"@trueplan/forecast-components": major
+---
+
+[select] updates Select, Option, and OptionGroup to use Select from ariakit. Breaking change with OptionGroup removes the label prop, should now use OptionGroupLabel component as a child
```

Comment:
Can we callout the breaking change more? Maybe just add it on a line itself, and make breaking change call caps.

Reply:
fixed